### PR TITLE
Update ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Microsoft 365 Business Premium in Content search and core ediscovery

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6816

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ediscovery.md [hunk 1]
```diff
@@ -126,13 +126,16 @@ The following sections show the minimum subscription requirements for Content se
 - Microsoft 365 E1 subscription
 - Microsoft 365 G1 subscription
 - Microsoft 365 F1 or F3 subscription, or F5 Security add-on
+- Microsoft 365 Business Premium
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery.md:129]:
```suggestion
- Microsoft 365 Business Premium subscription
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery.md]:
```suggestion
- Microsoft 365 Business Premium subscription
```

File: microsoft-365/compliance/ediscovery.md [hunk 2]
```diff
@@ -126,13 +126,16 @@ The following sections show the minimum subscription requirements for Content se
...
 - Office 365 Education A1 subscription
 - Office 365 E1 subscription
 
+
 ### Core eDiscovery
 
 - Microsoft 365 E3 subscription
 - Microsoft 365 G3 subscription
+- Microsoft 365 Business Premium
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery.md]:
```suggestion
- Microsoft 365 Business Premium subscription
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Pull request merged**